addons/web/static/src/views/view_dialogs/export_data_dialog.js

```diff
@@ -4,9 +4,10 @@ import { CheckBox } from "@web/core/checkbox/checkbox";
 import { Dialog } from "@web/core/dialog/dialog";
 import { useService } from "@web/core/utils/hooks";
 import { fuzzyLookup } from "@web/core/utils/search";
+import { useDebounced } from "@web/core/utils/timing";
 import { useSortable } from "@web/core/utils/sortable";
```

Comment:
nit picking: try to keep imports alphabetically sorted -> utils/timing after utils/sortable